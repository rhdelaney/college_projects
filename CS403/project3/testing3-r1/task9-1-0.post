(println "trying 8 numbers")
(stream-display 8 (ramanujan))
(println)
