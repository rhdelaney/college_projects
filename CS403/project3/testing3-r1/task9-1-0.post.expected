trying 8 numbers
(1729 4104 13832 20683 32832 39312 40033 46683 ...)
